src/main/java/duke/Event.java

```diff
@@ -0,0 +1,59 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * This class is a subclass of Task.
+ * @author Sim Jun Heng
+ * @version CS2103T AY21/22 Sem 2
+ */
+public class Event extends Task {
+
+    private LocalDate date;
+
+    /**
+     * Assigns desc, date and done to this instance.
+     *
+     * @param desc the task description
+     * @param date the date of the event
+     * @param isDone the current completion status of the task.
+     */
```

Comment:
According to the coding standard stated here [link](https://se-education.org/guides/conventions/java/intermediate.html), it states that all javadoc should have the following form where each @params line ends with a fullstop. IsDone has a fullstop at the end so perhaps you overlooked the previous 2.